de.bund.bfr.knime.fsklab.nodes/plugin.xml

```diff
@@ -52,6 +52,9 @@
      <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.joiner.JoinerNodeFactory" />
      <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.editor.FSKEditorJSNodeFactory" />
      
+     <!-- Please deprecate the next extension point definition before release -->
+     <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.fskdbview.FSKDBViewNodeFactory"/>

```

Comment:
We should mark the node as deprecated while is developed. We can always undeprecate it to debug it or include it in a local test build.